app/components/@settings/tabs/data/forms/EditDataSourceForm.tsx

```diff
@@ -478,13 +478,12 @@ export default function EditDataSourceForm({
           )}
         </div>
 
-        <div>
-          <ResourceAccessMembers
-            resourceScope="DATA_SOURCE"
-            resourceId={selectedDataSource.dataSource.id}
-            onAddMembers={onAddMembers}
-          />
-        </div>
+        <ResourceAccessMembers
+          resourceScope="DATA_SOURCE"
```

Comment:
It would be better if we created an enum for the available resource names, so we avoid using literal strings. 
Also one small remark, the "resourceScope" argument name is a bit misleading since this is just a name of the resource, not an actual scope.